diff --git a/node_modules/angular-2-dropdown-multiselect/bundles/angular-2-dropdown-multiselect.umd.js b/node_modules/angular-2-dropdown-multiselect/bundles/angular-2-dropdown-multiselect.umd.js
index 75018db..824ce31 100644
--- a/node_modules/angular-2-dropdown-multiselect/bundles/angular-2-dropdown-multiselect.umd.js
+++ b/node_modules/angular-2-dropdown-multiselect/bundles/angular-2-dropdown-multiselect.umd.js
@@ -601,6 +601,7 @@
                 else {
                     this.model = [];
                 }
+                this.prevModel = this.model;
             };
         /**
          * @param {?} fn
diff --git a/node_modules/angular-2-dropdown-multiselect/esm2015/dropdown/dropdown.component.js b/node_modules/angular-2-dropdown-multiselect/esm2015/dropdown/dropdown.component.js
index 65a37e7..00adb6b 100644
--- a/node_modules/angular-2-dropdown-multiselect/esm2015/dropdown/dropdown.component.js
+++ b/node_modules/angular-2-dropdown-multiselect/esm2015/dropdown/dropdown.component.js
@@ -274,6 +274,7 @@ export class MultiselectDropdownComponent {
         else {
             this.model = [];
         }
+        this.prevModel = this.model;
     }
     /**
      * @param {?} fn
diff --git a/node_modules/angular-2-dropdown-multiselect/esm5/dropdown/dropdown.component.js b/node_modules/angular-2-dropdown-multiselect/esm5/dropdown/dropdown.component.js
index 35d672f..aec21a0 100644
--- a/node_modules/angular-2-dropdown-multiselect/esm5/dropdown/dropdown.component.js
+++ b/node_modules/angular-2-dropdown-multiselect/esm5/dropdown/dropdown.component.js
@@ -327,6 +327,7 @@ var MultiselectDropdownComponent = /** @class */ (function () {
         else {
             this.model = [];
         }
+        this.prevModel = this.model;
     };
     /**
      * @param {?} fn
diff --git a/node_modules/angular-2-dropdown-multiselect/fesm2015/angular-2-dropdown-multiselect.js b/node_modules/angular-2-dropdown-multiselect/fesm2015/angular-2-dropdown-multiselect.js
index b3e5bc6..091f675 100644
--- a/node_modules/angular-2-dropdown-multiselect/fesm2015/angular-2-dropdown-multiselect.js
+++ b/node_modules/angular-2-dropdown-multiselect/fesm2015/angular-2-dropdown-multiselect.js
@@ -448,6 +448,7 @@ class MultiselectDropdownComponent {
         else {
             this.model = [];
         }
+        this.prevModel = this.model;
     }
     /**
      * @param {?} fn
diff --git a/node_modules/angular-2-dropdown-multiselect/fesm5/angular-2-dropdown-multiselect.js b/node_modules/angular-2-dropdown-multiselect/fesm5/angular-2-dropdown-multiselect.js
index cdb9278..b94e8fd 100644
--- a/node_modules/angular-2-dropdown-multiselect/fesm5/angular-2-dropdown-multiselect.js
+++ b/node_modules/angular-2-dropdown-multiselect/fesm5/angular-2-dropdown-multiselect.js
@@ -548,6 +548,7 @@ var MultiselectDropdownComponent = /** @class */ (function () {
         else {
             this.model = [];
         }
+        this.prevModel = this.model;
     };
     /**
      * @param {?} fn
